src/Adapter/DbSelect.php

```diff
@@ -145,7 +147,10 @@ protected function getSelectCount()
 
         $countSelect = new Select;
 
-        $countSelect->columns([self::ROW_COUNT_COLUMN_NAME => new Expression('COUNT(1)')]);
+        $countSelect->columns([
+            self::ROW_COUNT_COLUMN_NAME => new Expression('COUNT(1)'),
+            strtolower(self::ROW_COUNT_COLUMN_NAME) => new Expression('COUNT(1)')
```

Comment:
Will this work on all databases? Will some databases throw an error if one of the columns does not exist, and/or return a count of zero in that scenario?

Reply:
currently tested and worked in mysql

Reply:
I tested in PostgreSQL and it works:

```
test=# select COUNT(1) as C, COUNT(1) as c from tbl_user;
 c | c 
---+---
 4 | 4
(1 row)
```

Reply:
I tried with both uppercase in PostgreSQL as well, and it works:

```sql
test=# select COUNT(1) as C, COUNT(1) as C from tbl_user;
 c | c 
---+---
 4 | 4
(1 row)
```

Reply:
Just tried in SQLite3 as well, and work with both uppercase and lowercase 👍 

```sql
➜  ~ sqlite3
SQLite version 3.32.3 2020-06-18 14:00:33
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> create table tbl_user(email primary key not null, name varchar(255));
sqlite> insert into tbl_user values('samsonasik@gmail.com', 'Abdul Malik Ikhsan');
sqlite> select COUNT(1) as C, COUNT(1) as C from tbl_user;
1|1
sqlite> select COUNT(1) as C, COUNT(1) as c from tbl_user;
1|1

```

Reply:
Does the database query really need to be extended? Can we really ensure for all DBMS that it works or that there are no performance problems? I think it introduced more questions / problems.

Why not check the result for the default column name in upper case and as an alternative in lower case?

Reply:
@froschdesign  updated. should be ok now.